pylsp/plugins/rope_autoimport.py

```diff
@@ -0,0 +1,211 @@
+import logging
```

Comment:
```suggestion
# Copyright 2022- Python Language Server Contributors.

import logging
```

Reply:
Fixed in 76f68de3a9b56c9f3ee014bee0da0bd141f8547a